pkg/consensus/consensus.go

```diff
@@ -280,6 +285,9 @@ events:
 			s.dbft.OnTransaction(tx)
 		case b := <-s.blockEvents:
 			s.handleChainBlock(b)
+			if s.lastTimestamp < b.Timestamp {
+				s.lastTimestamp = b.Timestamp
+			}
```

Comment:
Don't we want to move `lastTimestamp`'s check to `s.handleChainBlock` in order to perform the same check 6 lines below?

Reply:
Fixed.